packages/smithy-http/src/smithy_http/aio/crt.py

```diff
@@ -174,13 +174,13 @@ def on_response(
                     kind=FieldPosition.HEADER,
                 )
 
-        self._response_future.set_result(
-            AWSCRTHTTPResponse(
-                status=status_code,
-                fields=fields,
-                body=self._body,
+            self._response_future.set_result(
+                AWSCRTHTTPResponse(
+                    status=status_code,
+                    fields=fields,
+                    body=self._body,
+                )
```

Comment:
This seems to still be indented so that it's now in the except block

Reply:
Good catch - was testing this on top of the event signing PR changes and messed up moving the changes back.